### PR TITLE
feat: update dependabot to new wildcard syntaxt

### PR DESCRIPTION
**Pull request by jorgepiloto**
Solves for #418 by implementing the new dependabot syntax. This version of dependabot is still in beta mode.

### CODE REVIEW

**Comment by RobPasMue** [comment]
Is it ready for review? I got notified

File: .github/dependabot.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ updates:
       - "dependencies"
 
   - package-ecosystem: "github-actions"
-    directory: "/"
+    directories: "**/*"
```
Comment (jorgepiloto) [inline .github/dependabot.yml:14]:
```suggestion
    directories: "!(requirements)/*"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: 2
 updates:
   - package-ecosystem: "pip"
-    directory: "/requirements"
+    directories: "/requirements/*"
```
Comment (jorgepiloto) [inline .github/dependabot.yml:4]:
```suggestion
    directories: 
      - "/requirements/*"
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ updates:
       - "dependencies"
 
   - package-ecosystem: "github-actions"
-    directory: "/"
+    directories: "!(requirements)/*"
```
Comment (jorgepiloto) [inline .github/dependabot.yml:14]:
```suggestion
    directories: 
      - "!(requirements)/*"
```

File: .github/dependabot.yml [hunk 4]
```diff
@@ -1,7 +1,8 @@
 version: 2
 updates:
   - package-ecosystem: "pip"
-    directory: "/requirements"
+    directories: 
+      - "/requirements/*"
```
Comment (jorgepiloto) [inline .github/dependabot.yml:5]:
```suggestion
      - "**/*"
```

File: .github/dependabot.yml [hunk 5]
```diff
@@ -11,7 +12,8 @@ updates:
       - "dependencies"
 
   - package-ecosystem: "github-actions"
-    directory: "/"
+    directories: 
+      - "!(requirements)/*"
```
Comment (jorgepiloto) [inline .github/dependabot.yml:16]:
```suggestion
      - "**/*"
```

File: .github/dependabot.yml [hunk 6]
```diff
@@ -11,7 +12,8 @@ updates:
       - "dependencies"
 
   - package-ecosystem: "github-actions"
-    directory: "/"
+    directories:
+      - "**/*"
     schedule:
```
Comment (RobPasMue) [inline .github/dependabot.yml:17]: Shouldn't only the "directories" directive be applied here? And not in the "pip" section, consequently

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 6 above)
Comment (jorgepiloto) [reply .github/dependabot.yml:17]: Yeah, that's a good point. Let me restore the lines for the pip ecosystem.

Comment (RobPasMue) [reply .github/dependabot.yml:17]: Great thanks!

**Review by SMoraisAnsys — APPROVED** [review]
LGTM !

**Review by MaxJPRey — APPROVED** [review]

**Comment by RobPasMue** [comment]
Feel free to merge @jorgepiloto

**Pull request merged**